repo/packages/W/weavescope-probe/0/config.json

```diff
@@ -0,0 +1,22 @@
+{
+  "type": "object",
+  "description": "Properties of Weave Scope probe package",
+  "properties": {
+    "weavescope": {
```

Comment:
The thing is that it doesn't make sense to have multiple instances of it, so I haven't templatised the names... Thanks a lot for the advice, I'll set do like you said, didn't see that in the docs.